insights/specs/default.py

```diff
@@ -548,6 +548,39 @@ def semid(broker):
     ovirt_engine_ui_log = simple_file("/var/log/ovirt-engine/ui.log")
     ovirt_engine_boot_log = simple_file("/var/log/ovirt-engine/boot.log")
     ovirt_engine_console_log = simple_file("/var/log/ovirt-engine/console.log")
+    ovs_vsctl_list_br = simple_command("/usr/bin/ovs-vsctl list-br")
+
+    @datasource(ovs_vsctl_list_br)
+    def ovs_bridge_mac_table_entry_count(broker):
+        """
+        Get the count of MAC entries present in the table.
+
+        Typical output of command ovs-appctl fdb/show <bridge-name>::
+
+            # ovs-appctl fdb/show br-int
```

Comment:
Hey, any reason you're not returning the entire table of information described here? That would be more generally useful.

Reply:
@csams - As explained by @xiangce, since the command output contains MAC addresses which is a sensitive data, it was decided to get the MAC count for a bridge in the client slide only as the plugin needed only the MAC count per bridge.
If it is allowed to collect MAC address, I'll enhance the parser as per suggestions.

Reply:
@csams - it is better to collect the whole table including MAC address. Just want to confirm with you if MAC is PII. Are we allowed to collect MAC address?